impeller/renderer/backend/vulkan/context_vk_unittests.cc

```diff
@@ -222,5 +222,16 @@ TEST(ContextVKTest, WarmUpFunctionCreatesRenderPass) {
                         "vkCreateRenderPass") != functions->end());
 }
 
+TEST(ContextVKTest, FatalMissingValidations) {
+  EXPECT_DEATH(const std::shared_ptr<ContextVK> context =
```

Comment:
Are you getting a warning about running this in a multithreaded environment?  This works by forking and catching signals which won't work reliably on multi threaded processes.

Reply:
Not running locally

Reply:
Is there a cmd line flag I can test this locally with?

Reply:
The tests that you execute before your test are going to matter.  If ever there is any test that can spawn a thread and not join it, like creating a thread pool, that run before this test it will be a problem.  I'd just try running the test runner with every test enabled.  Maybe try enabling the test shuffle a few times and seeing if it prints out the error.  I almost guarantee there is at least one test that is leaking threads though.  That's what I was seeing with the golden test runner at least.

Reply:
gtest parallel runs multiple gtest processes, if one or more of those process spawn a subprocess that shouldn't cause any issues. The problems we've had usually stem from reading/writing to the file system.

Reply:
The problem isn't from running the tests in multiple processes, it comes from the `signal` function.  Which thread will handle a signal is undefined.  That's what's being used to catch the death of the process.  I'm pretty sure that's why googletest prints out the warning.  If you don't see the warning and want to give it a shot that sounds good to me.  We should add a comment to the test in case it ever starts flaking.